cocos/ui/UIPageView.h

```diff
@@ -76,7 +76,15 @@ class CC_GUI_DLL PageView : public Layout
     enum class TouchDirection
     {
         LEFT,
-        RIGHT
+        RIGHT,
+        UP,
```

Comment:
These constants are used internally, so I need to move them into private section.